File: pyg_sellergoods_interface/src/main/java/com/pinyougou/sellergoods/service/BrandService.java
package com.pinyougou.sellergoods.service;

import com.pinyougou.enity.PageResult;
import com.pinyougou.pojo.TbBrand;

import java.util.List;

public interface BrandService {
    //查找所有品牌
    public List<TbBrand> findAll();
    //条件查询+分页
    PageResult findPage(TbBrand brand, Integer pageNum, Integer pageSize);
    //添加
    void add(TbBrand brand);
    //通过id查找
    TbBrand findOne(Long id);
    //修改
    void update(TbBrand brand);
    //删除
    void delete(Long[] ids);
}

File: pyg_manager_web/src/main/webapp/js/service/brandService.js
//自定义服务
app.service('brandService',function ($http) {
    ////定义查询所有品牌列表的方法
    this.findAll=function () {
        return $http.get("../brand/findAll.do");
    }
    //条件查询和分页展示
    this.search=function (searchEntity,pageNum,pageSize) {
        return $http.post("../brand/search.do?pageNum="+pageNum+"&pageSize="+pageSize,searchEntity);
    }
    //添加和修改
    this.add=function (entity) {
        return $http.post("../brand/add.do",entity);
    }
    //修改
    this.update=function (entity) {
        return $http.post("../brand/update.do",entity);
    }
    //通过id查找
    this.findOne=function (id) {
        return $http.get("../brand/findOne.do?id="+id);
    }
    //删除
    this.delete=function (selectIds) {
        return $http.get("../brand/delete.do?ids="+selectIds);
    }


})